hw/xbox/xbox.c

```diff
@@ -52,7 +52,9 @@
 
 #define MAX_IDE_BUS 2
 
-// XBOX_TODO: Should be passed in through configuration
+int eeprom_fd = -1;
```

Comment:
This will have to be in `XboxMachineState`, in case someone creates more than one `xbox` machine.